siriuspy/siriuspy/devices/intlkctrl.py

```diff
@@ -8,7 +8,9 @@
 class ASPPSCtrl(_Device):
     """Accelerator PPS Control."""
 
-    PROPERTIES_DEFAULT = ('TunAccessRemainingWaitTime-Mon')
+    _properties = (
+        'TunAccessWaitTimeLeft-Mon',
```

Comment:
merge problem here: correct is `'TunAccessRemainingWaitTime-Mon'`

Reply:
@anacso17 , we have changed it to `TunAccessWaitTimeLeft-Mon`

Reply:
Hi @xresende, did you change in the IOC? I think the previous name is more appropriate...

Reply:
But indeed, there is problem with the class constant too: PROPERTIES_DEFAULT

Reply:
> Hi @xresende, did you change in the IOC? I think the previous name is more appropriate...

the IOC is using `TunAccessWaitTimeLeft-Mon`.